documentation/manage/updating-your-extension.md

```diff
@@ -0,0 +1,216 @@
+| `update_link`             | `string` (Optional)               | A link to the XPI file containing this version of the add-on. This must be an HTTPS URL, or an `update_hash` must be provided to verify it.                                                                                                                                                                                                                                                                                                               |
+| `update_hash`             | `string` (Optional)               | A cryptographic hash of the file pointed to by `update_link`. This must be provided if `update_link` is not a secure URL. If present, this must be a string beginning with either `sha256:` or `sha512:`, followed by the hexadecimal-encoded hash of the matching type.                                                                                                                                                                                  |
+| `update_info_url`         | `string` (Optional)               | A link to an HTML file containing information about the update.                                                                                                                                                                                                                                                                                                                                                                                           |
+| `multiprocess_compatible` | `bool` (Optional) (default: true) | If false, this add-on requires compatibility shims to run in a [multi-process Firefox](https://developer.mozilla.org/docs/Mozilla/Firefox/Multiprocess_Firefox) environment.                                                                                                                                                                                                                                                                              |
+| `applications`            | `object` (Optional)               | An object containing application-specific compatibility information. Each property must contain an application object, as described below. The only application currently supported is `gecko`, which includes Firefox, and all other applications built on the same runtime. If this property is omitted, support for Gecko is assumed. Otherwise, if this property is defined, it must contain a `gecko` property, or the update entry will be ignored. |
+
+{% endcapture %}
+{% include modules/table.html
+	content=table
+%}
+
+<!-- END: Table -->
+
```

Comment:
Needs h3 for Application Objects with line for `addons[*].updates[*].applications.gecko` beneath it.

Reply:
Add anchor link to this section.